cmd/fs/cp_test.go

```diff
@@ -0,0 +1,221 @@
+package fs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/databricks/cli/libs/filer"
+)
+
+func TestCp_concurrencyValidation(t *testing.T) {
```

Comment:
Can you please add this as an acceptance test instead? That's the primary way we test CLI commands in this repo.